examples/pytorch/contrastive-image-text/run_clip.py

```diff
@@ -559,7 +559,11 @@ def filter_corrupt_images(examples):
         trainer.save_metrics("eval", metrics)
 
     # 11. Write Training Stats and push to hub.
-    kwargs = {"finetuned_from": model_args.model_name_or_path, "tasks": "contrastive-image-text-modeling"}
+    finetuned_from = model_args.model_name_or_path
+    # If from a local directory, don't set `finetuned_from`
```

Comment:
It would be better to describe why here in the comment. It's clear from the code this is what's happening but not the motivation

Reply:
Changed it to

```
# If from a local directory, don't set `finetuned_from` as this is required to be a valid repo. id on the Hub.
```